install.php

```diff
@@ -0,0 +1,109 @@
+<?php
+// Verify that we are allowed to start installing
+
+if(file_exists('./application/config/database.php'))
+{
+	exit("It seems that you have already installed this package. If you mean to re-install, please delete 'database.php' located in './application/config/'");
+}
+
+echo '<link href="./assets/css/bootstrap.min.css" rel="stylesheet" media="screen">';
+echo '<link href="./assets/css/main.css" rel="stylesheet">';
+echo '<title>Store Control Panel Setup</title>';
+
+
+if (isset($_POST['submit']))
+{
+	processInfo();
+} else {
+	requestInfo();
+}
+
+function requestInfo()
+{
+	// This is the way novices will be installing the script.
+	// That is why I only ask them for critical details, the rest can remain default.
+	echo <<<HTML
+		<div class="container">
+			<h1>Setup</h1>
+			<p>In order for this package to function, you must supply access details to a MySQL database, and a secret encryption key (this can be random).</p>
+			<form action="install.php" method="post">
+				<p><label for="host">Hostname:</label><input type="text" name="host">
+				<label for="user">Username:</label><input type="text" name="user">
+				<label for="pass">Password:</label><input type="text" name="pass">
+				<label for="name">Database:</label><input type="text" name="name"></p>
+				<p><label for="key">Encryption key:</label><input type="text" name="key"></p>
+				<input type="submit" name="submit" value="Submit">
+			</form>
+			<hr>
+			<footer>
+				<p>&copy; 2013-2015 Store Plugin Developed by <a href="https://github.com/alongubkin/" target="_blank"><strong>alongub</strong></a> and <a href="https://github.com/Drixevel/" target="_blank"><strong>Drixevel</strong></a>. Code by <a href="https://github.com/Arrow768/" target="_blank"><strong>Arrow768</strong></a>. Design by <strong>Bor3d Gaming</strong>.
+				</p><p> Go to <a href="https://github.com/SourceMod-Store/store-webpanel/issues/" target="_blank"><strong>Github</strong></a> to report Bugs and request Features.</p>
+			</footer>
+		</div>
+HTML;
+}
+
+function processInfo() 
+{
+	// Import the SQL stuffs first. If these go wrong, no harm done.
+	// This is going to hurt your eyes.
+	mysql_connect($_POST['host'],$_POST['user'],$_POST['pass']) or die('Error connecting to MySQL. No harm done, just start the installation again.');
+	mysql_select_db($_POST['name']) or die('Error selecting MySQL database: ' . mysql_error());
+	
```

Comment:
You should perform a check if the store tables are already imported and throw a Info message if they are not.

(Just check if the store_users table exists)